server.go

```diff
@@ -1,10 +1,10 @@
 package main
 
 import (
+	urlexpander "./lib"
```

Comment:
hm ... strange. it seems it worked for me because i was compiling (go build -o server server.go) [outside of any `GOPATH` work space](https://golang.org/cmd/go/#hdr-Import_path_syntax).